python/src/multi_agent_orchestrator/agents/bedrock_llm_agent.py

```diff
@@ -33,13 +33,13 @@ def __init__(self, options: BedrockLLMAgentOptions):
         if options.client:
             self.client = options.client
         else:
-            if options.region:
-                self.client = boto3.client(
+
+            self.client = boto3.client(
                     'bedrock-runtime',
-                    region_name=options.region or os.environ.get('AWS_REGION')
-                )
-            else:
-                self.client = boto3.client('bedrock-runtime')
+                    region_name=options.region or os.environ.get('AWS_REGION'))
```

Comment:
Now you are forcing the condition of having `options.region` or the `AWS_REGION` env. It is certainly my fault for not understanding the entire codebase, but if that is not the explicit intention, you are probably creating a regression.